index.html

```diff
@@ -60,219 +65,122 @@
           </nav>
           <section class="container headline-section d-flex flex-column">
```

Comment:
The application you are creating should be similar to the provided design. That includes fonts, colors, any background image, font-size etc. You can personalize the content but the basic design guidelines should be followed. Accordingly, kindly update this section as per the design provided. Consider using a background image for the section and a gradient image for the text.
| Your design  | Provided design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/192083778-2d6a0cc1-b1eb-4114-8894-702695502e1f.png) | ![image](https://user-images.githubusercontent.com/33268927/192083750-f770c62b-d605-4e4d-9fc4-63dbbbbb9257.png) |

Reply:
- [ ] @yetemegn-telaye great work implementing the changes requested, however, your design as demonstrated below doesn't have a similar layout also you don't have a background image, kindly fix it.
![image](https://user-images.githubusercontent.com/61408860/192110778-f86fbef5-d509-4143-8530-fc190039be86.png)